packages/rolldown/tests/fixtures/resolve/custom-modules/_config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineTest } from 'rolldown-tests'
+import path from 'node:path'
+
+export default defineTest({
+  config: {
+    resolve: {
+      modules: [path.join(__dirname, 'custom_node_modules')],
```

Comment:
I believe it should be

```suggestion
      modules: ['custom_node_modules'],
```